compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -1431,9 +1453,15 @@ static LogicalResult setAttentionIntrinsicBasedVectorDistributionConfig(
                               /*rhsType=*/vElementType,
                               /*accType=*/f32Type};
 
-  GPUMMAHeuristicSeeds pvMatmulSeeds = {/*bestSubgroupCountPerWorkgroup=*/4,
-                                        /*bestMNTileCountPerSubgroup=*/4,
-                                        /*bestKTileCountPerSubgroup=*/4};
+  int64_t bestMNTileCountPerSubgroup = 4;
+  if (clUseExperimentalAttentionHeuristics) {
+    bestMNTileCountPerSubgroup = 8;
```

Comment:
I'm a bit concerned about this. This should depend on the number of heads i.e. K1/N. K1/N are always static for realistic workloads (if they are dynamic, we should probably not even care about them). It's okay to start with, I do know we should be using more MN tiles, so this is okay, but a comment would be good here.